### ISSUE TITLE
A JavaScript error occurred in the main process while opening Visual Studio Code

### ISSUE THREAD

**Issue by fekfoq259** [microsoft/vscode#75221]
https://github.com/Microsoft/AL/issues/74
https://github.com/Microsoft/vscode/issues/11378
https://github.com/microsoft/vscode/issues/10616
https://github.com/Microsoft/vscode/issues/46159

- VSCode Version: x64-1.35.0
- OS Version: Windwos 10 1903

Recent I have this "old" problem more than 4 times.
I try uninstall reinstall than problem disappear but after a while it continuing happen. （Even I don't install any extension）
Any suggetion? (fully uninstall and reinstall)

Does this issue occur when all extensions are disabled?: Yes
![image](https://user-images.githubusercontent.com/45431701/59230484-2d5dd400-8c10-11e9-96e1-09651cc5399a.png)

It can't normally uninstall.
![image](https://user-images.githubusercontent.com/45431701/59230625-a0ffe100-8c10-11e9-8795-6ef3b07d831a.png)

**Comment by brentil** [comment]
I can confirm we're having the same issue again too.  SCCM deployed updated VSCode 1.35.0 to all of our developer's systems.  Developers are using it just fine for a while, close out a work space, and this starts happening and they can no longer open VSCode at all.  We use the system level install of the 64bit version on Windows 10 1809 & 1903.  The solution I found to get them working again was to go to each machine and manually update it to VSCode 1.35.1 and then they could start working again.  So far I've had to do this to 4 developer's systems.
